### ISSUE TITLE
`Bug` "In 2 days" is wrongly displayed in Czech

### ISSUE THREAD

**Issue by mbisak** [Blackymas/NSPanel_HA_Blueprint#1695]
### TFT Version

4.2.4

### Blueprint Version

4.2.4

### Panel Model

NSPanel EU Model

### What is the bug?

"In 2 days" is wrongly displayed in Czech

### Steps to Reproduce

"In 2 days" is displayed as "Za dni". Missing number 2. It should be displayed as "Za 2 dni". In 3 day is correctly shown as "Za 3 dni".

**Comment by edwardtfn** [comment]
`Za 2 dny` or `Za 2 dni`?

**Comment by mbisak** [comment]
Hi, both are correct according to the institute for Czech language. 

Plural/accusative https://prirucka.ujc.cas.cz/en/?slovo=den&Hledej=Search

Other translations are Za X dny and  they are correct, SO put there Za 2 dny.

Martin

Sent from Nine

________________________________
Od: Edward Firmo ***@***.***>
Odesláno: pátek 19. ledna 2024 15:39
Komu: Blackymas/NSPanel_HA_Blueprint
Kopie: mbisak; Author
Předmět: Re: [Blackymas/NSPanel_HA_Blueprint] `Bug` "In 2 days" is wrongly displayed in Czech (Issue #1695)

Za 2 dny or Za 2 dni?

**Comment by edwardtfn** [comment]
Fixed on `dev` and will be included on v4.2.5.

**Comment by edwardtfn** [comment]
Those are all the strings for translation. Can you find something else that needs changing?

```yaml
    cs:     # Czech
      weekdays:
        mon: Pondělí
        tue: Úterý
        wed: Středa
        thu: Čtvrtek
        fri: Pátek
        sat: Sobota
        sun: Neděle
      weekdays_short:
        mon: Po
        tue: Út
        wed: St
        thu: Čt
        fri: Pá
        sat: So
        sun: Ne
      months:
        jan: Leden
        feb: Únor
        mar: Březen
        apr: Duben
        may: Květen
        jun: Červen
        jul: Červenec
        aug: Srpen
        sep: Září
        oct: Říjen
        nov: Listopad
        dec: Prosinec
      months_short:
        jan: Led
        feb: Úno
        mar: Bře
        apr: Dub
        may: Kvě
        jun: Čvn
        jul: Čvc
        aug: Srp
        sep: Zář
        oct: Říj
        nov: Lis
        dec: Pro
      relative_day:
        today: Dnes
        tomorrow: Zítra
        in_2_days: Za 2 dny
        in_3_days: Za 3 dny
        in_4_days: Za 4 dny
      climate:
        states:
          "on": Zapnuto
          "off": Vypnuto
        heat: Topení
      please_confirm: Prosím, potvrďte
      unavailable: Nedostupné
      unknown: Neznámý
      no_name: Bez názvu
      alarm:
        home: Doma
        away: Pryč
        night: Noc
        vacation: Dovolená
        bypass: Obcházet
        disarm: Zneškodnit
      settings:
        reboot: Restart
        sleep: Spánek
        brightness: Jas
        bright: Světlý
        dimming: Ztmavení
        dim: Tmavý
      meridiem:
        am: dop.
        pm: odp.
```

**Comment by edwardtfn** [comment]
Done!

**Comment by edwardtfn** [comment]
We just released a patch with solves this: [v4.2.5 - Celebrating 1000 Stars with Enhanced Functionality and UI Improvements](https://github.com/Blackymas/NSPanel_HA_Blueprint/releases/tag/v4.2.5)
Could you please update your system and let me know if the issue was solved?
